intelmq/bots/collectors/shadowserver/collector_reports_api.py

```diff
@@ -36,6 +39,8 @@ class ShadowServerAPICollectorBot(CollectorBot, HttpMixin, CacheMixin):
     api_key = None
     secret = None
     types = None
+    reports = None
```

Comment:
Please describe this parameter in docs/user/bots.rst

Reply:
Added.